tests/test_views.py

```diff
@@ -630,3 +631,16 @@ def test_get(self) -> None:
         self.assertContains(response, subject.title)
         self.assertContains(response, subject.description, count=1)
         self.assertContains(response, question.title, count=1)
+
+
+class Test404PageNotFound(TestCase):
```

Comment:
@pjdelport I didn't know this, all the tutorials kept referring to it :)
Thank you the heads up